Reduce parentdirs we use, parentdirs are used for ordering
Included upstream
diff -u --new-file --recursive rpm-5.1.9_vanilla/lib/depends.c rpm-5.1.9_no-parentdirs/lib/depends.c
--- rpm-5.1.9_vanilla/lib/depends.c	2009-04-12 19:46:17.000000000 +0000
+++ rpm-5.1.9_no-parentdirs/lib/depends.c	2009-06-13 15:21:43.504999639 +0000
@@ -2257,7 +2257,7 @@
 #define isAuto(_x)	((_x) & _autobits)
 
 /*@unchecked@*/
-static int slashDepth = 100;	/* #slashes pemitted in parentdir deps. */
+static int slashDepth = 2;	/* #slashes pemitted in parentdir deps. */
 
 static int countSlashes(const char * dn)
 	/*@*/
